package_manifest.yaml

```diff
@@ -416,6 +417,7 @@ foreman_client_packages:
       - f27-foreman-client-nightly
   hosts:
     katello-host-tools:
+      package_base_dir: 'packages/katello/'
```

Comment:
Just a thought: should we move this to `packages/client`? Doesn't have to be part of this PR.